--- libspiff-0.7.3/configure.ac.orig	2007-09-14 21:56:40.000000000 +0200
+++ libspiff-0.7.3/configure.ac	2007-09-15 14:45:09.396937254 +0200
@@ -87,14 +87,6 @@
 
 
 
-# Append $prefix as a fallback at the very end
-if test "x${prefix}" != "xNONE"; then
-	WORKING_PREFIX=${prefix}
-else
-	WORKING_PREFIX=${ac_default_prefix}
-fi
-LDFLAGS="${LDFLAGS} -L${WORKING_PREFIX}/lib"
-CPPFLAGS="${CPPFLAGS} -I${WORKING_PREFIX}/include"
 
 
 
